interfaces/HO-Portal/src/global.scss

```diff
@@ -97,6 +97,15 @@ ion-button {
       --padding-end: 8px;
     }
   }
+
+  .validation-message {
+    min-height: 36px;
+
+    ion-note {
+      font-weight: bold;
+      font-size: 12px;
```

Comment:
Would using a `<small>`+`<strong>`-element also work visually?

So we don't need the specific CSS here.